archetypes/data-playground.md

```diff
@@ -19,6 +19,33 @@ sdks: "Add Available SDKs Here and Separate Multiple SDKs With A Comma"
 limits: "Add API Limit Information Here, If Available"
 
 ---
+{{% data-playground-wrapper %}} 
+
+{{% data-playground-code-tabs %}}
+{{% data-playground-code-tab tabIndex="python" name="Python"  %}}
+```python
+      def example(name):
+      print(f"example, {name}!")
+```
+{{% /data-playground-code-tab %}}
+
+{{% data-playground-code-tab tabIndex="go" name="Go"  %}}
+
+```go {linenos=table,hl_lines=[1,"3-4"]}
+package main
+import "fmt"
+func main() {
+    fmt.Println("hello world")
+}
+```
+
+{{% /data-playground-code-tab %}}
+{{% /data-playground-code-tabs %}}
+
+{{% /data-playground-wrapper %}}
+
+{{% data-playground-content %}}
 
 <!-- Add content for data playground here, including a table for data products if available -->
 
+{{% /data-playground-content %}}
```

Comment:
Could you explain the difference between this shortcode and the one on line 47? I see that there's a `/` on this line and not on 47, but there appears to only be one shortcode for the content.

Reply:
Never mind. I looked up the Hugo docs and see what's happening. I was used to seeing other shortcodes without the `%`.